### PR TITLE
[FTLIS042022] Chloé Faurie

### PR DESCRIPTION
**Pull request by chloe4E**
solved all iterations

I tried the #7, seems OK though it does not pass all the tests - not sure why :)

### CODE REVIEW

**Review by webLisbonTA — COMMENTED** [review]
Great job on finishing all the mandatory iterations!

File: src/movies.js [hunk 1]
```diff
@@ -1,35 +1,246 @@
...
+//         score: 9.7
+//       }
+//     ])
+// );
 
 // Iteration 4: Drama movies - Get the average of Drama Movies
-function dramaMoviesScore() {}
+function dramaMoviesScore(arr) {
+  let dramas = arr.filter((element) => element.genre.includes('Drama'));
+  return scoresAverage(dramas);
```
Comment (webLisbonTA) [inline src/movies.js:86]: Amazing job using the previous function to ease your work!

File: src/movies.js [hunk 2]
```diff
@@ -1,35 +1,246 @@
...
+    let adate = a.year;
+    let bdate = b.year;
+    let rv = adate - bdate;
+    if (rv === 0) {
+      rv = a.title.localeCompare(b.title);
+    }
+    return rv;
+  });
+  return sortedArr;
+}
```
Comment (webLisbonTA) [inline src/movies.js:142]: This solution is correct but you could do it with the same logic but a bit shorter:
```
function orderByYear(movies) {
  const ordered = [...movies].sort((a, b) => {
    if (a.year !== b.year) {
      return a.year - b.year;
    }
    return a.title.localeCompare(b.title);
  })
  return ordered;
}
```

Comment (chloe4E) [reply src/movies.js:142]: smart

**Pull request closed without merging**